constructor.js

```diff
@@ -0,0 +1,147 @@
+var allMonths = [];
+
+var january = new Month('january');
```

Comment:
Since you're pushing these to the `allMonths` array anyhow, it doesn't look like you would need to give these new instances variable references. 

You could also put the month strings in an array, and then iterate over the array to create a new instance of the month with each value of the array. 

```javascript
for (month in months) {
  new Month(months[month]);
}
```